nectar/downloaders/threaded.py

```diff
@@ -264,7 +264,7 @@ def _fetch(self, request):
             if netloc in self.failed_netlocs:
                 raise SkipLocation()
 
-            _logger.debug("Attempting to connect to {url}.".format(url=request.url))
+            _logger.info("Attempting to connect to {url}.".format(url=request.url))
```

Comment:
I think if I were a user, I'd want this to stay at debug. One statement to tell me whether each download worked or not is helpful. Adding this doubles the number of statements getting logged, and I don't think it adds enough value to justify the noise.

What do you think about that?